client/bindings/nodejs/test/client/offlineSigningExamples.spec.ts

```diff
@@ -97,8 +97,8 @@ describe('Offline signing examples', () => {
 
         const unlock = await offlineClient.signatureUnlock(
             secretManager,
-            preparedTx.inputsData[0],
             hexToBytes(txEssenceHash),
+            preparedTx.inputsData[0].chain!,
```

Comment:
I really don't like that, we need to find another solution, maybe returning an error instead, idk

Reply:
But this is just a test

Reply:
Indeed